lib/src/presentation/widgets/movie_list_page_widgets/now_playing_movies_page_view.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:core';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_project/src/core/utils/extensions.dart';
+import '../../../core/utils/styles.dart';
+import '../../../domain/entities/genre.dart';
+import '../../../domain/entities/movie.dart';
+import '../../bloc/movie_list_page_bloc.dart';
+import '../../pages/detail_movie_page.dart';
+
+class nowPlayingMoviesPageView extends StatelessWidget {
```

Comment:
Rename to `NowPlayingMoviesPageView` (start with a capital letter)

Reply:
Oops, right! Done!